src/main/java/de/tum/in/www1/artemis/service/metis/conversation/ChannelService.java

```diff
@@ -240,4 +248,131 @@ public void unarchiveChannel(Long channelId) {
         conversationService.notifyAllConversationMembersAboutUpdate(updatedChannel);
     }
 
+    /**
+     * Create a channel for a lecture
+     *
+     * @param lecture     the lecture to create the channel for
+     * @param channelName the name of the channel
+     * @return the created channel
+     */
+    public Channel createLectureChannel(Lecture lecture, @NotNull String channelName) {
+        Channel channelToCreate = new Channel();
+        channelToCreate.setName(channelName);
+        channelToCreate.setIsPublic(true);
+        channelToCreate.setIsAnnouncementChannel(false);
+        channelToCreate.setIsArchived(false);
```

Comment:
Do these values need to be specified here everytime? Aren't there some usable default values?

Reply:
Yes, we have to set these, there is no default.